src/redux/actions/embedding/loadEmbedding.js

```diff
@@ -38,7 +33,7 @@ const loadEmbedding = (experimentId, embeddingType) => async (dispatch, getState
   const body = {
     name: 'GetEmbedding',
     type: embeddingType,
-    config: embeddingConfig,
+    config: methodSettings[embeddingType],
```

Comment:
@marcellp I updated the body according to the request to fit the API specification.